### ISSUE TITLE
time stamps are out of sync

### ISSUE THREAD

**Issue by tylerzika** [standardnotes/android#133]
When I re-encrypted all of my notes on my macOS app, it told me to sign out of all existing signed in applications or risk getting my data corrupted. So I signed out of my android app, then signed back in. All of my notes are there, but they all have the same time stamp now (the time I logged back in), but the times stamps on my macOS app are install intact.

**Comment by mobitar** [comment]
Yeah unfortunately when you re-encrypt all your notes, it updates the "modified at" date of the item. On Android, items are sorted by that date, so it won't necessarily be what you expect them to be (or what you actually modified them at). On desktop, you can sort by created at (default), which will look normal.

Still trying to figure out a solution for this.
